Dockerfile

```diff
@@ -1,14 +1,14 @@
-FROM golang:1.18.1-alpine3.15 AS gobuild-base
-RUN apk add --no-cache \
-	git \
-	make
+FROM mcr.microsoft.com/oss/go/microsoft/golang:1.18-fips-cbl-mariner2.0 AS gobuild-base
+RUN tdnf clean all && tdnf install -y \
+git \
+make
 
 FROM gobuild-base AS acr-cli
 WORKDIR /go/src/github.com/Azure/acr-cli
 COPY . .
 RUN make binaries && mv bin/acr /usr/bin/acr
 
-FROM alpine:3.15
-RUN apk --update add ca-certificates
+FROM mcr.microsoft.com/cbl-mariner/base/core:2.0.20221122
```

Comment:
I think we can use the stable tag `mcr.microsoft.com/cbl-mariner/base/core:2.0`

Reply:
Done